### ISSUE TITLE
Mark features active by default as on

### ISSUE THREAD

**Issue by kontur** [justvanrossum/fontgoggles#57]
The way I am reading the grey/green/red status of the feature labels "grey" communicates using "whatever the renderer does by default" — however, it would be useful to actually know what features are on.

'rlig' as well as the GPOS features are on by default, but the current UI does not communicate this. I think there should be a difference between the color indicators of _your choice_ and communicating _what is active in the renderer_.

**Comment by justvanrossum** [comment]
Unfortunately HarfBuzz (the underlying shaping/layout engine) does not expose this information, at least not as far as I know.

Also: it depends on the script of the entered text, so it's not a generally static thing.

Indeed, "gray" means: whatever the shaper decides to be on, and the colors are user overrides.

**Comment by justvanrossum** [comment]
Perhaps I'm wrong, though.

@khaledhosny, @behdad: does HarfBuzz have an API to query which features will be on/off by default, given a certain text string?

**Comment by khaledhosny** [comment]
No, there isn't currently. The list does not change that much, so checking
HarfBuzz source code and hard-coding the lists would be good enough.

**Comment by justvanrossum** [comment]
Hm, given how this varies per script it seems a bit of a rabbit hole, though. (I say this after perusing the HB source code.)

**Comment by khaledhosny** [comment]
I think having a HarfBuzz API for this is good idea. There is a HarfBuzz issue https://github.com/harfbuzz/harfbuzz/issues/561 for this, may be you want to comment and revive the discussion.
